### ISSUE TITLE
listFields won't show fields unless also specified in displayFields

### ISSUE THREAD

**Issue by alexjreich** [nxus/web#53]
It seems like these two should be totally separate, rather than listFields being a subset of displayFields.  Use case is including things in the list for filtering purposes that we may not want editable through the form.

**Comment by loppear** [comment]
Yeah, these are a bit tangled because the model attributes are filtered by displayFields even on list view.
👍

**Comment by davidkellerman** [comment]
Just to be clear, there's the interaction between `displayFields` and `listFields`, but also `EditController` inherits from `ViewController`, which means it too is getting an attribute list from `_modelAttributes()` that's filtered through `displayFields`, both directly and in `defaultContext()`.
